azure-pipelines-full-tests.yml

```diff
@@ -92,3 +72,99 @@ jobs:
         pythonVersion: '$(python.version)'
         profile: '2018-03-01-hybrid'
         fullTest: true
+
+- job: AutomationFullTestPython36ProfileLatest
+  displayName: Automation Full Test Python36 Profile Latest
+  timeoutInMinutes: 9999
+  strategy:
+    maxParallel: 8
+    matrix:
+      8_1:
```

Comment:
Use more meaningful name

Reply:
Use instance$idx instead of unmeaningful name.
![image](https://user-images.githubusercontent.com/18628534/164188219-519b49e9-6153-4972-9331-5ceab6dc8854.png)